Sources/Workspace/UserToolchain.swift

```diff
@@ -114,24 +117,27 @@ public final class UserToolchain: Toolchain {
         return toolPath
     }
 
-    private static func findTool(_ name: String, envSearchPaths: [AbsolutePath]) throws -> AbsolutePath {
+    private static func findTool(_ name: String, envSearchPaths: [AbsolutePath], searchPathsOnly: Bool) throws -> AbsolutePath {
```

Comment:
how about we flip the condition to say "useXcrun"? we can also make that a macOS only API signature which would make a nicer API overall because then can make sure it is only called with that flag where appropriate

Reply:
I think flipping sounds reasonable, but different signatures seem like a bit overkill for private methods?

Reply:
true